VTEX - VTEX Shipping Network API.json

```diff
@@ -3,7 +3,7 @@
+      "description": "[VTEX Shipping Network](https://vtex.com/br-pt/shipping-network/) is a solution that connects carriers and merchants for deliveries at better freight costs and smarter logistics operations.\r\n\r\n>ℹ️ **VTEX Shipping Network** is exclusive to Brazil.\r\n\r\nThe tracking process has two main parts:\r\n- **Notification:** The [carrier](https://help.vtex.com/en/tutorial/carriers-on-vtex--7u9duMD5UQa2QQwukAWMcE) is notified that a package is being dispatched.\r\n- **Tracking:** The package being delivered by the carrier is tracked from the beginning of the delivery until its final location.\r\n\r\nThe carrier's apps have to integrate with the hub, which is connected to other VTEX systems. The carrier is required to develop [VTEX IO](https://developers.vtex.com/docs/guides/vtex-io-documentation-what-is-vtex-io) apps and associate them with specific endpoints.\r\n\r\n>ℹ️ To learn more, see our [integration guide](https://developers.vtex.com/docs/guides/vtex-shipping-network).\r\n\r\n## VTEX Shipping Network API Index\r\n\r\n### Notification\r\n\r\n- `POST` [Notify carrier with app](https:\/\/developers.vtex.com\/docs\/api-reference\/vtex-shipping-network-api#post-\/-app_name-\/v-app_version-\/-account-\/-workspace-\/notify)\r\n\r\n### Tracking\r\n\r\n- `POST` [Tracking events with app](https:\/\/developers.vtex.com\/docs\/api-reference\/vtex-shipping-network-api#post-\/-app_name-\/v-app_version-\/-account-\/-workspace-\/tracking)\r\n\r\n## Common parameters\r\n\r\n| **Parameter name** | **Description** | **Type** |\r\n| :---: | :--- | :--- |\r\n| `{{accountName}}` | Name of the VTEX account. Used as part of the URL. | Server variable. |\r\n| `{{environment}}` | Environment to use. Used as part of the URL. The default value is `vtexcommercestable`. | Server variable. |\r\n| `X-VTEX-API-AppKey` | Unique identifier of the [application key](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys). | Authentication header. Must be used together with `X-VTEX-API-AppToken`. Not necessary when using `VtexIdclientAutCookie`. |\r\n| `X-VTEX-API-AppToken` | Secret token of the [application key](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys). | Authentication header. Must be used together with X-VTEX-API-AppKey. Not necessary when using `VtexIdclientAutCookie`. |\r\n| `VtexIdclientAutCookie` | [User token](https://developers.vtex.com/docs/guides/api-authentication-using-user-tokens), valid for 24 hours. | Authentication header. Not necessary when using `X-VTEX-API-AppKey` and `X-VTEX-API-AppToken`. |",
```

Comment:
The following change you made to the description seems to have switched the order of information.

> The carrier's apps have to integrate with the hub, which is connected to other VTEX systems. The carrier is required to develop [VTEX IO](https://developers.vtex.com/docs/guides/vtex-io-documentation-what-is-vtex-io) apps and associate them with specific endpoints.

In my opinion, it should be in this order: Carriers have to develop IO apps -> these IO apps will integrate with the hub, which is connected to other VTEX systems.

Reply:
Suggestion accepted!